src/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": [
+    "@integreat-app/eslint-config-jest",
+    "@integreat-app/eslint-config-flow",
+    "@integreat-app/eslint-config-babel"
+  ],
+  "rules": {
+    "object-curly-spacing": ["off", "never"]
```

Comment:
Shouldn't this rule rather be added to `eslint-config-flow`?

Reply:
See here: https://issues.integreat-app.de/browse/LINT-10
This needs to be fixed yes.